advanced/list-of-log-names.md

```diff
@@ -1,72 +1,76 @@
-| TEMPMUTE\_REMOVE   | Emitted when a tempmute is removed/has expired            |
-| TEMPBAN\_REMOVE    | Emitted when a tempban is removed/has expired             |
-| AUDIT\_KICK        | Emitted when a user is kicked but not by the bot          |
-| AUDIT\_BAN\_ADD    | Emitted when a user is banned, but not by the bot         |
-| ROLE\_ADD          | Emitted when a role is added through the `role` command   |
-| ROLE\_REMOVE       | Emitted when a role is removed through the `role` command |
-| SELFROLE\_ADD      | Emitted when a selfrole is added                          |
-| SELFROLE\_REMOVE   | Emitted when a selfrole is removed                        |
-| RAID\_CLEANUP      | Emitted when a raid is cleaned up                         |
-| INFRACTION\_DELETE | Emitted when an infraction is deleted                     |
-
-### Messages
-
-| Type                 | Description                               |
-| -------------------- | ----------------------------------------- |
-| CHANNEL\_CLEAN       | Emitted when a channel is cleaned         |
-| MESSAGE\_DELETE      | Emitted when a message is deleted         |
-| MESSAGE\_EDIT        | Emitted when a message is edited          |
-| MESSAGE\_PIN\_ADD    | Emitted when a message is pinned          |
-| MESSAGE\_PIN\_REMOVE | Emitted when a pinned message is unpinned |
-
-### Members
-
-| Type       | Description                            |
-| ---------- | -------------------------------------- |
-| USER\_JOIN | Emitted when an user joins the server  |
-| USER\_LEFT | Emitted when an user leaves the server |
-
-### Debug
+| Name      | Emitted when     |
+| --------- | ---------------- |
+| USER_JOIN | Someone joins |
+| USER_LEFT | Someone leaves |
+
+## Messages
+
+This category is for things having to do with messages.
+
+| Name                | Emitted when     |
+| ------------------- | ---------------- |
+| CHANNEL_CLEAN       | Someone cleans all the messages from a channel |
+| MESSAGE_BULK_DELETE | Someone bulk deletes a set of messages |
+| MESSAGE_DELETE      | Anyone deletes a message |
+| MESSAGE_EDIT        | Anyone edits a message |
+| MESSAGE_PIN_ADD     | A message gets pinned |
+| MESSAGE_PIN_REMOVE  | A message gets unpinned |
+
+## Debug
 
 {% hint style="info" %}
-This category is for debugging purposes. Logs in this category are meant to aid in fixing errors that may prevent the bot for functioning correctly
+This category is for debugging purposes. Logs in this category are meant to aid in fixing errors that may prevent the bot from functioning correctly.
+
 {% endhint %}
 
-| Type                    | Description                              |
-| ----------------------- | ---------------------------------------- |
-| REACTION\_ROLE_\__DEBUG | Debug for reaction roles related actions |
+| Name                | Emitted when     |
+| ------------------- | ---------------- |
+| REACTION_ROLE_DEBUG | Someone uses a reaction role |
```

Comment:
Incorrect, this one is for when someone triggers a reaction role but something goes wrong. It's meant to help the server admins understand what's wring with the server's reaction role configuration.